chrome/content/zotero/zoteroPane.js

```diff
@@ -59,6 +59,7 @@ var ZoteroPane = new function()
 	this.clearItemsPaneMessage = clearItemsPaneMessage;
 	this.viewSelectedAttachment = viewSelectedAttachment;
 	this.reportErrors = reportErrors;
+	this.setDynamicTooltip = setDynamicTooltip;
```

Comment:
We haven't used this convention in many, many years. If it's a public function, it should just be `this.functionName = function () {` below.